src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/layout/HmrcLayoutParams.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.govukfrontend.views.viewmodels.layout
+
+import uk.gov.hmrc.govukfrontend.views.Aliases.BackLink
+
+// TODO name?
+case class HmrcLayoutParams(
+  pageTitle: Option[String] = None,
```

Comment:
I'm just thinking maybe we should have value classes for **pageTitle** and **nonce** to keep all parameters typed while we have the opportunity during this refactor

Reply:
> **TODO - agree structure / naming of new layout class** I personally believe the structure you have now is well-defined as for the name of the layout I just cant think of anything good.

maybe it should be `uk.gov.hmrc.govukfrontend.views.viewmodels.hmrcstandardlayout.Params` to make it clear it's not a hmrc-frontend component but models for the helper?